Backpack-SwiftUI/SectionHeader/Classes/BPKSectionHeader.swift

```diff
@@ -0,0 +1,79 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+public struct BPKSectionHeader: View {
+    let title: String
+    let description: String?
+    let style: Style
+    let button: BPKButton?
+    let accessibilityLabel: String
+
+    public init(
+        title: String,
+        description: String? = nil,
+        style: Style = .default,
+        accessibilityLabel: String
+    ) {
+        self.title = title
+        self.description = description
+        self.style = style
+        self.button = nil
+        self.accessibilityLabel = accessibilityLabel
+    }
+
+    public init(
+        title: String,
+        description: String? = nil,
+        style: Style = .default,
+        accessibilityLabel: String,
+        @ViewBuilder button: () -> BPKButton
+    ) {
+        self.title = title
+        self.description = description
+        self.style = style
+        self.accessibilityLabel = accessibilityLabel
+        self.button = button()
+    }
+
+    public var body: some View {
+        HStack {
+            VStack(alignment: .leading) {
+                BPKText(title, style: .heading3)
+                    .foregroundColor(style == .default ? .textPrimaryColor : .textOnDarkColor)
```

Comment:
```suggestion
                    .foregroundColor(style == .default ? .textPrimaryColor : .textOnDarkColor)
                    .accessibilityAddTraits(.isHeader)
```

We can add the `.isHeader` trait to the title, as it will make navigation eassier and we expect this component to behave as such. 

Not 100% sure if we want both title & subtitle combined, and then behaving as a heading, or just the title.